index.html

```diff
@@ -0,0 +1,31 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="./style.css">
+    <title>Real </title>
+</head>
+
+<body>
+    <header id="header"> 
+        <h1>Kano-Bruda Real Estate</h1>
+        <img src="./vector-illustration-cool-detailed-red-260nw-94498447.jpg" id="header-img" alt="">
+    <nav id="nav-bar">
+        <ul>
+            <a class="nav-link" href="#Overview"><li>Overview</li></a>
+                    <a class="nav-link" href="#view_Homes"><li>Locations</li></a>
+                    <a class="nav-link" href="#homws"><li>Customers</li></a>
+                    
+        </ul>
+    </nav>
+    </header>
+    
+    <form  id="form" action="">
+        email: <input id="email" type="text">
```

Comment:
It is best if you include a label tag to provide a proper label for your input field as opposed to just plain text. The label tag is good semantic html and useful for users with screen readers

Reply:
Nice job being polite, giving reasons for the requested change, and phrasing your comments as suggestions